Sources/MessagingPushFCM/MessagingPushFCM.swift

```diff
@@ -92,15 +92,15 @@ public class MessagingPushFCM: MessagingPushFCMInstance {
         _ request: UNNotificationRequest,
         withContentHandler contentHandler: @escaping (UNNotificationContent) -> Void
     ) -> Bool {
-        messagingPush.didReceive(request, withContentHandler: contentHandler)
+        (messagingPush as MessagingPushInstance).didReceive(request, withContentHandler: contentHandler)
```

Comment:
In iOS v2, I made some refactors that no longer require this casting. But for v1 with the way the code is today, it still requires this casting to be done.